Remotion/Web/Development.WebTesting.TestSite.Shared/WebButtonTest.aspx.cs

```diff
@@ -16,16 +16,20 @@
 // 
 using System;
 using System.Web.UI.WebControls;
+using Remotion.ServiceLocation;
+using Remotion.Web.Development.WebTesting.TestSite.Infrastructure;
 using Remotion.Web.ExecutionEngine;
 
-namespace Remotion.Web.Development.WebTesting.TestSite
+namespace Remotion.Web.Development.WebTesting.TestSite.Shared
 {
   public partial class WebButtonTest : WxePage
   {
     protected override void OnInit (EventArgs e)
     {
       base.OnInit(e);
 
+      MyWebButtonWithIcon.Icon.Url = ResolveUrl(this.ResolveImageResource("SampleIcon.gif"));
```

Comment:
```suggestion
      MyWebButtonWithIcon.Icon.Url = this.ResolveImageResource("SampleIcon.gif");
```

Shouldn't this be enough?